src/fastfield/alive_bitset.rs

```diff
@@ -55,7 +55,7 @@ impl AliveBitSet {
         AliveBitSet::from(readonly_bitset)
     }
 
-    /// Opens a delete bitset given its file.
+    /// Opens an alive bitset given its file.
```

Comment:
The doc in this file is written as if we were tombstoning, but the code seems to say we're "inverse tombstoning".

Reply:
yes thanks, that was inverted recently